rules/DowngradePhp73/Rector/FuncCall/SetCookieOptionsArrayToArgumentsRector.php

```diff
@@ -118,24 +112,20 @@ private function composeNewArgs(FuncCall $funcCall): array
     {
         $this->highestIndex = 1;
 
-        if (! $this->argsAnalyzer->isArgsInstanceInArgsPositions($funcCall->args, [0, 1, 2])) {
```

Comment:
`if ($funcCall->isFirstClassCallable()) {` should be checked early

Reply:
Here it's included in the `count($node->getArgs())` check